nucypher/config/constants.py

```diff
@@ -53,7 +54,6 @@
 A Learner learning about the GLOBAL_DOMAIN will learn about all nodes.
 A Teacher serving the GLOBAL_DOMAIN will teach about all nodes.
 """
-GLOBAL_DOMAIN = b'GLOBAL_DOMAIN'
```

Comment:
The docstring above should be updated as `GLOBAL_DOMAIN` is not used anymore.

Reply:
:heavy_check_mark: